BUILD.gn

```diff
@@ -1061,6 +1061,7 @@ if (is_mac) {
       ":electron_app_plist",
       ":electron_app_resources",
       ":electron_fuses",
+      "//base",
```

Comment:
Did you try this?

```suggestion
      "//base:tracing_buildflags",
```

Reply:
in agreement with @zcbenz - i'd prefer we fix this with as small a scope as possible. Depending on _all_ of `base` for a more specific failure feels unncecessary.

Reply:
`//base:tracing_buildflags` is transitive dependency because of `base::FilePath` usage and would hide the underlying issue.